src/types.mligo

```diff
@@ -296,22 +281,10 @@ type initial_data =
   ; config_data : initial_config_data
   }
 
-type decision_lambda = decision_lambda_input -> decision_lambda_output
+type decision_callback = decision_callback_input -> decision_callback_output
 
 type config =
-  { proposal_check : propose_params * contract_extra -> unit
-  // ^ A lambda used to verify whether a proposal can be submitted.
-  // It checks 2 things: the proposal itself and the amount of tokens frozen upon submission.
-  // It allows the DAO to reject a proposal by arbitrary logic and captures bond requirements
-  ; rejected_proposal_slash_value : proposal * contract_extra -> nat
-  // ^ When a proposal is rejected, the value that the proposer gets back can be slashed.
-  // This lambda returns the amount to be slashed.
-  ; decision_lambda : decision_lambda
```

Comment:
_I'll mention this here just to pick a place to leave this comment_

I see that there are several mentions of this being a "lambda" yet:
- in documentation: `README.md`, `docs/specification.md`, `docs/registry.md`, etc.
- in Haskell modules, mostly in comments and some test names
- in LIGO code, again mostly in comments and some local variables

Reply:
Have renamed everything now.